terms/custom.scss

```diff
@@ -1,8 +1,12 @@
-nav.navbar,
-footer {
-  display: none;
-}
+// The '@docusaurus/plugin-content-blog' plugin id terms is set in docusaurus.config.js
+
+.plugin-id-terms {
```

Comment:
1. this is unclear why `.plugin-id-terms` matches the plugin in config
2. `config.ts`

Reply:
This is the previously existed code that was removed in my last PR. I just move them back in.

I thought you'd have more context because if I remember correctly this `.plugin-id-terms` trick was initially added by you...

Reply:
Discussed offline and improved code comments